src/engage/enrollments/enrollment.action.ts

```diff
@@ -0,0 +1,96 @@
+import {ZapierItem} from '../../types'
+import {createEnrollment, stopEnrollment} from '../common'
+import {createActionDetails} from '../../utils/operations'
+import {userSearches, userTriggers} from '../../users/keys'
+import {enrollmentActions, enrollmentSearches, enrollmentTriggers, sequenceSearches, sequenceTriggers} from '../keys'
+import EnrollmentResource from './enrollment.resource'
+import {leadSearches, leadTriggers} from '../../lead/keys'
+
+export const CreateEnrollmentAction: ZapierItem = {
+    key: enrollmentActions.createEnrollmentAction,
+    noun: 'Enrollment',
+    display: {
+        label: 'Create Enrollment',
+        description: 'Creates a new enrollment in catalog. Requires Reach subscription.'
+    },
+    operation: {
+        resource: EnrollmentResource.key,
+        inputFields: [
+            {
+                key: 'sequence_id',
+                label: 'Sequence',
+                type: 'integer',
+                required: true,
+                dynamic: `${sequenceTriggers.sequenceListDropdown}.id.name`,
+                search: `${sequenceSearches.sequenceSearch}.id`,
+            },
+            {
+                key: 'sell_resource',
+                label: 'Sell Resource',
+                children: [
+                    {
+                        key: 'resource_id',
+                        label: 'Lead',
+                        helpText: 'Zendesk Sell Lead that will be enrolled.',
+                        required: true,
+                        type: 'integer',
+                        dynamic: `${leadTriggers.leadListDropdown}.id.name`,
+                        search: `${leadSearches.leadSearchOrCreate}.id`
+
+                    },
+                    {
+                        key: 'resource_type',
+                        label: 'Resource Type',
+                        choices: ['lead'],
+                        type: 'string',
+                        required: true
+                    }
+                ]
+            },
+            {
+                key: 'actor_id',
+                label: 'Owner',
+                helpText: 'Zendesk Sell User that will be an actor for enrollment.',
```

Comment:
That will be the user that emails will be sent by and tasks will be created for.